src/services/CalculatorProcess.js

```diff
@@ -1,51 +1,52 @@
-let memory = [];
-let result = "";
-function calculatorProcess(content,visor) {
-	
-	const numbersOperators = () => String(eval((result)));
-	
-	memory += content;
+let memory = "";
+let sLength;
 
-	if (memory.includes("=")) {
+function CalculatorProcess(content, visor) {
 
-		result = memory.slice(0,-1);
+	const firstInteraction = ["%", "/", "*", "+", "-", "S", "C", "="];
 
-		if (memory.includes("+")) {
-			memory = numbersOperators();
-		}
+	const conversion = (string) => String(eval(string));
 
-		if (memory.includes("-")) {
-			memory = numbersOperators();
-		}
+	memory += content;
 
-		if (memory.includes("*")) {
-			memory = numbersOperators();
-		}
+	if ((firstInteraction.includes(memory[0]) && memory.length < 2) || memory.includes("C")) {
```

Comment:
1 - this `(` is useless
2 - it's not clear what this condition does: `firstInteraction.includes(memory[0]) && memory.length < 2`.

Reply:
ahh I guess I got it what this condition does :)

there is other way to check this, maybe if you write a test first you can find out :)

Reply:
i created this condition with this `memory.length < 2` to don't get conflict with the operators condition(that operators are not allowed at first position)

Reply:
> ahh I guess I got it what this condition does :)
> 
> there is other way to check this, maybe if you write a test first you can find out :)

ok, then i'll resolve this

Reply:
done